### PR TITLE
Added Youtube timelapse.

### PR DESCRIPTION
**Pull request by ryanfox1985**

### CODE REVIEW

File: _plugins/youtube_timelapse.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+source: https://github.com/ryanfox1985/OctoPrint-Youtube-Timelapse
+archive: https://github.com/ryanfox1985/OctoPrint-Youtube-Timelapse/archive/master.zip
+tags:
+- youtube
+- timelapse
+- oAuth
+- google
+
+screenshots:
+- url: https://raw.githubusercontent.com/ryanfox1985/OctoPrint-Youtube-Timelapse/master/screenshots/configuration_step7.png
```
Comment (jneilliii) [inline _plugins/youtube_timelapse.md:19]: image needs to be included in the PR, not linked to your github repo for GDPR concerns, place them in `/assets/img/plugins/youtube_timelapse/configuration_step7.png` and update link.

File: _plugins/youtube_timelapse.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  alt: Main plugin screenshot
+  caption: Main plugin screenshot
+
+featuredimage: https://raw.githubusercontent.com/ryanfox1985/OctoPrint-Youtube-Timelapse/master/screenshots/configuration_step7.png
```
Comment (jneilliii) [inline _plugins/youtube_timelapse.md:23]: image needs to be included in the PR, not linked to your github repo for GDPR concerns, place them in `/assets/img/plugins/youtube_timelapse/configuration_step7.png` and update link.

**Review by jneilliii — CHANGES_REQUESTED** [review]
Not sure how I feel about you using my screenshots...lol, but you need to update your plugin's repo init file for python 3 only compatibility [here](https://github.com/ryanfox1985/OctoPrint-Youtube-Timelapse/blob/312af77164589d6e6bb164e5826f95dcb420c890/octoprint_youtube_timelapse/__init__.py#L264). 

It's also a good idea to update the line [here](https://github.com/ryanfox1985/OctoPrint-Youtube-Timelapse/blob/312af77164589d6e6bb164e5826f95dcb420c890/setup.py#L62) to be like below, to prevent python 2 users from being able to manually install as well.
```
additional_setup_parameters = {"python_requires": ">=3, <4"}
```

Also, please see the two notes below in regard to your image assets, they need to be included as part of your PR.

**Comment by ryanfox1985** [comment]
@jneilliii thanks for the feedback, I corrected both points:

- My repository with Python requirement (good catch!) => https://github.com/ryanfox1985/OctoPrint-Youtube-Timelapse/commit/e8b6fbff925cbbcb8ebd30a2e461b7aaee6268c8
- Image added to the PR.

About the images, I'm sorry but I took yours for my commodity, I thought it was no problem to use these resources to bring a new opensource plugin.

Regards.

**Comment by cp2004** [comment]
Please enable issues on your repository so people can get in touch for problems. They're not enabled for forks by default but can be enabled in the settings.

**Comment by jneilliii** [comment]
I agree with @cp2004 here, always best to have an issue tracker enabled to allow people to have somewhere to go for problems, otherwise they come here and post issues that we have no way of knowing how to fix.

**Comment by ryanfox1985** [comment]
oh thanks!, I thought the issues tracker enabled was a default setting.

**Pull request merged**